jenkins/Jenkinsfile-blossom.premerge

```diff
@@ -230,6 +232,78 @@ pipeline {
                 }
             }
         } // end of Premerge Test
+
+        // TODO: support parallel testing for different spark versions
+        stage('Databricks Test') {
+            when {
+                beforeAgent true
+                expression {
+                    !skipped
+                }
+            }
+
+            agent {
+                kubernetes {
+                    label "premerge-test-${BUILD_TAG}"
+                    cloud 'sc-ipp-blossom-prod'
+                    yaml "$pluginPremerge"
+                    workspaceVolume persistentVolumeClaimWorkspaceVolume(claimName: "${PVC}", readOnly: false)
+                    customWorkspace "${CUSTOM_WORKSPACE}"
```

Comment:
I would assume we don't require resources like the pre-merge running on blossom here. This could be a simple container w/ some cpu and memory, so it could be easier to be scheduled by blossom